### PR TITLE
Give leaderboard have history charts

### PR DESCRIPTION
**Pull request by IntoThe1337ness**
##### Description

this PR let leaderboard have history charts

##### Refers/Fixes

there is thread on twitter and townsquare about this. crates competitivness.

##### Testing

i tested local (See Below Comment)

##### Other Notes

we have to give the new cleanup DB script deploy first.  Give data a few Week to AccumuLte.

### CODE REVIEW

**Comment by IntoThe1337ness** [comment]
> 10  main    5432 down   postgres /var/lib/postgresql/10/main /var/log/postgresql/postgresql-10-main.log

Not sure what this means or why build fails.  Cannot Find anything similar to my PR in the build Logs.

**Comment by IntoThe1337ness** [comment]
```
from django.utils import timezone
from marketing.models import LeaderboardRank
import random

LeaderboardRank.objects.all().delete()

start = timezone.now()
for i in range(0, 10):
    date = start - timezone.timedelta(days=i*7)
    for j in range(0, 10):
        lbr_kwargs = {
        'count': j,
        'active': i == 0,
        'amount': (((10 - i) * 10) + j * 10) + random.randint(0,1000),
        'rank': j,
        'leaderboard': 'weekly_earners',
        'github_username': f"1337haxxor{j}",
        'product': 'all',
        'created_on': date,
        }
        LeaderboardRank.objects.create(**lbr_kwargs)
        print(i, j)
```

Here is a screenCapture of What this looks like using the above mock data @octavioamu :

screencapture-localhost-8000-leaderboard-2020-02-19-19_47_43

File: app/retail/templates/leaderboard.html [hunk 1]
```diff
@@ -126,6 +130,13 @@ <h1 class="font-title-lg font-weight-semibold mt-2">{% trans "None found" %}</h1
                     </p>
                   </div>
                 {% else %}
+
+              {% if cht %}
+              <div class="row no-gutter">
+                <div id='container' style="width:100%;"> Chart will be rendered here </div>
+              </div>
```
Comment (thelostone-mc) [inline app/retail/templates/leaderboard.html]: could we indent this by 1

File: app/retail/templates/leaderboard.html [hunk 2]
```diff
@@ -126,6 +130,13 @@ <h1 class="font-title-lg font-weight-semibold mt-2">{% trans "None found" %}</h1
                     </p>
                   </div>
                 {% else %}
+
+              {% if cht %}
+              <div class="row no-gutter">
+                <div id='container' style="width:100%;"> Chart will be rendered here </div>
```
Comment (thelostone-mc) [inline app/retail/templates/leaderboard.html]: All the html file use double quotes so could we update that

File: app/retail/templates/leaderboard.html [hunk 3]
```diff
@@ -3,6 +3,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+  <script src="{% static "v2/js/lib/jquery.js" %}"></script>
```
Comment (thelostone-mc) [inline app/retail/templates/leaderboard.html]: wouldn't jquery be loaded twice in this page in this case ?

File: app/retail/templates/leaderboard.html [hunk 4]
```diff
@@ -202,6 +213,7 @@ <h1 class="font-title-lg font-weight-semibold mt-2">{% trans "None found" %}</h1
       $("img").unveil(200);
     });
   </script>
+  <script src='https://code.highcharts.com/highcharts.js'></script>
```
Comment (thelostone-mc) [inline app/retail/templates/leaderboard.html]: singlequotes -> double quotes

**Comment by danlipert** [comment]
Same as the other branch, lets have the base be the master branch - thanks!

File: app/retail/templates/leaderboard.html (hunk 3 above)
Comment (IntoThe1337ness) [reply app/retail/templates/leaderboard.html]: OK FIXED

**Comment by IntoThe1337ness** [comment]
OK THIS PR FIXES CODE REVIEW AND BASE BRANCH https://github.com/gitcoinco/web/pull/6076

**Pull request closed without merging**